package/rtcm3_protocol/src/framer_rtcm3.c

```diff
@@ -131,7 +132,10 @@ uint32_t framer_process(void *state, const uint8_t *data, uint32_t data_length,
       if (count > available) {
         count = available;
       }
-
+      if (count > RTCM3_BUFFER_SIZE - s->buffer_length) {
+        piksi_log(LOG_INFO, "RTCM buffer full\n");
+        count = RTCM3_BUFFER_SIZE - s->buffer_length;
```

Comment:
Oops, if we ever end up here, then we next read the buffer to its full capacity and then the logic reaches a dead end as it wants to read more data but cannot. Besides we don't seem to hit this condition even when the original problem exhibits itself.

Reply:
Let's try again then..